packages/image_picker/image_picker/pubspec.yaml

```diff
@@ -16,19 +16,18 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_plugin_android_lifecycle: ^2.0.0-nullsafety.2
-  image_picker_platform_interface: ^2.0.0-nullsafety
+  flutter_plugin_android_lifecycle: ^2.0.0
+  image_picker_platform_interface: ^2.0.0
 
 dev_dependencies:
-  video_player: ^2.0.0-nullsafety.7
   flutter_test:
     sdk: flutter
   integration_test:
     path: ../../integration_test
   mockito: ^5.0.0-nullsafety.7
-  pedantic: ^1.8.0
+  pedantic: ^1.10.0
   plugin_platform_interface: ^1.0.3
```

Comment:
```
Checking project image_picker...
  info - The library 'package:plugin_platform_interface/plugin_platform_interface.dart' is legacy, and should not be imported into a null safe library at test/image_picker_test.dart:10:8 - (import_of_legacy_library_into_null_safe)
```

Reply:
This dep. needs to be set as ">=1.0.0 <3.0.0", right @stuartmorgan?

Reply:
Yes

Reply:
this will get fixed after merging https://github.com/flutter/plugins/pull/3580